docker/package_managers/install_pkgs.bzl

```diff
@@ -41,7 +41,8 @@ users will write something like:
 
 def _generate_install_commands(tar, installation_cleanup_commands):
     return """
-tar -xvf {tar}
+tar -xf {tar}
+export DEBIAN_FRONTEND=noninteractive
```

Comment:
Would it be difficult to attempt to add a test that validates this is set? This seems like an important addition. I'm not sure how things worked before this.

Reply:
There is no difference in the built image before and after the change. The only difference is the log message of "bazel build".

If `DEBIAN_FRONTEND` is not set, `dpkg` will try multiple frontends and eventually fallback to `Teletype`:

```
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.28.1 /usr/local/share/perl/5.28.1 /usr/lib/x86_64-linux-gnu/perl5/5.28 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.28 /usr/share/perl/5.28 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype
```

After we explicitly set `DEBIAN_FRONTEND`, `dpkg` won't print those logs.

PS: You can see these logs by running
```shell
docker run --rm debian:buster-slim bash -c 'apt-get update && apt-get install -y locales'
```